### PR TITLE
Deploy helper scripts within the image deploy directory

### PR DESCRIPTION
**Pull request by iartemenko**
Signedoff-by: Iurii Artemenko <iurii_artemenko@epam.com>

### CODE REVIEW

File: recipes-domd/domd-image-weston/domd-image-weston.bbappend [hunk 1]
```diff
@@ -69,3 +69,10 @@ configure_versions_rcar() {
 python do_configure_append_rcar() {
     bb.build.exec_func("configure_versions_rcar", d)
 }
+
+do_install_append () {
+    local LAYERDIR=${TOPDIR}/../meta-xt-prod-gen3-test
+    find ${LAYERDIR}/doc -iname "u-boot-env*" -exec cp -f {} ${DEPLOY_DIR}/domd-image-weston/images/${MACHINE}-xt \; || true
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/domd-image-weston.bbappend]: Currently it is named u-boot-env-salvator-x-h3-m3.txt. But it contains nothing SoC specific. Could you rename it to u-boot-env-salvator-x.txt for example?

File: recipes-domd/domd-image-weston/domd-image-weston.bbappend [hunk 2]
```diff
@@ -69,3 +69,10 @@ configure_versions_rcar() {
...
+    find ${LAYERDIR}/doc -iname "mk_sdcard_image.sh" -exec cp -f {} ${DEPLOY_DIR}/domd-image-weston/images/${MACHINE}-xt \; || true
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-weston/domd-image-weston.bbappend]: I was thinking where to store this script. Taking into the account that we feed this script with the root images directory, would it be better to put script there?

File: recipes-domd/domd-image-weston/domd-image-weston.bbappend (hunk 1 above)
Comment (iartemenko) [reply recipes-domd/domd-image-weston/domd-image-weston.bbappend]: Done

Comment (iartemenko) [reply recipes-domd/domd-image-weston/domd-image-weston.bbappend]: Done

**Pull request merged**